### PR TITLE
Adds hackathon results to /results page

### PR DESCRIPTION
**Pull request by owocki**
##### Description

Adds hackathon results to /results page
https://github.com/gitcoinco/web/issues/4944

##### Testing

tested locallly:

Screen Shot 2019-08-25 at 4 30 58 PM

### CODE REVIEW

File: app/kudos/kudos.yaml [hunk 1]
```diff
@@ -5118,3 +5118,11 @@
   - Friends
   - Love
   image: 3box_butterfly.svg
+- name: DAI
```
Comment (owocki) [inline app/kudos/kudos.yaml]: i dont know why this is on my PR.  it already exists on master.

Comment (thelostone-mc) [reply app/kudos/kudos.yaml]: 😓  Is github UI messed up again

**Comment by owocki** [comment]
is this ok to merge?  looks like it was approved but somehow didnt make the release...

**Comment by danlipert** [comment]
@octavioamu does this look good to you?

**Pull request merged**